apps/api/schedule.py

```diff
@@ -169,6 +170,56 @@ def post(self, proposal_type):
         return [t.id for t in res]
 
 
+class C3VOCPublishingWebhook(Resource):
+    method_decorators = {"post": [_require_video_api_key]}
+
+    def post(self):
+        if not request.is_json:
+            abort(415)
+
+        payload = request.get_json()
+
+        try:
+            conference = payload["fahrplan"]["conference"]
+            proposal_id = payload["fahrplan"]["id"]
+        except KeyError:
+            abort(422)
+
+        if not payload["is_master"]:
+            # c3voc *should* only send us information about the master
+            # encoding getting published. Aborting early ensures we don't
+            # accidentially delete video information from the database.
+            abort(406)
```

Comment:
Although HTTP 403 is _usually_ authorization-related, in this case I think it might be the appropriate status code here.  We're denying the edit to the resource because we don't think it's allowable given the contents of the request.

```suggestion
            abort(403)
```

Out of interest: what would non-master videos be?  (would those be in-progress drafts / edits, for example?)

Reply:
A non-master would be something like a scaled-down version of the same video - these *should* have the same urls, but i would not vouch for that.

Regarding the http status code, i'm not sure either. My thought was "it's not acceptable here to do updates for non-master encodings", but i can also fully understand the thought of "it's not allowed to do that". Will give it a thought.